applications/GeoMechanicsApplication/tests/dsettlement/fully_saturated_column_uniform_load/ProjectParameters_stage2.json

```
@@ -0,0 +1,230 @@
+{
+  "problem_data": {
+    "problem_name"                    : "mesh",
+    "start_time"                      : 0.0,
+    "end_time"                        : 8640000.0,
+    "echo_level"                      : 1,
+    "parallel_type"                   : "OpenMP",
+    "number_of_threads"               : 1
+  },
+  "solver_settings": {
+    "solver_type"                     : "U_Pw",
+    "model_part_name"                 : "PorousDomain",
+    "domain_size"                     : 2,
+    "model_import_settings": {
+      "input_type"                    : "mdpa",
+      "input_filename"                : "mesh"
```

Here we can use the following to make sure we don't re-read the mesh (same holds for stage 3-5)
```suggestion
      "input_type"                    : "use_input_model_part"
```